client/my-sites/backup/backup-contents-page/index.tsx

```diff
@@ -0,0 +1,43 @@
+import { Card } from '@automattic/components';
+import { FunctionComponent } from 'react';
+import ActionButtons from 'calypso/components/jetpack/daily-backup-status/action-buttons';
+import cloudIcon from 'calypso/components/jetpack/daily-backup-status/status-card/icons/cloud-success.svg';
+import useGetDisplayDate from 'calypso/components/jetpack/daily-backup-status/use-get-display-date';
+import { useLocalizedMoment } from 'calypso/components/localized-moment';
+import './style.scss';
+
+interface OwnProps {
+	rewindId: number;
+}
+
+const BackupContentsPage: FunctionComponent< OwnProps > = ( { rewindId } ) => {
+	const getDisplayDate = useGetDisplayDate();
+	const moment = useLocalizedMoment();
+	const displayDate = getDisplayDate( moment.unix( rewindId ), false );
+
+	return (
+		<>
+			<div className="main backup-contents-page">
+				<Card className="card backupdaily-backup-status contents-page">
+					<div className="contents-page__header">
+						<div className="status-card__message-head">
+							<img src={ cloudIcon } alt="" role="presentation" />
+							<div className="status-card__hide-mobile">Backup contents from:</div>
+						</div>
+						<div className="status-card__title">{ displayDate }</div>
+						<ActionButtons
+							disabled={ false }
+							hasWarnings={ false }
+							isMultiSite={ false }
+							rewindId={ rewindId.toString() }
+							onClickClone={ null }
```

Comment:
The `onClickClone` prop seems to be mandatory on [`ActionButtons`](https://github.com/Automattic/wp-calypso/blob/f588238cdb6345a5dbe945fc1abb2f84b0455295/client/components/jetpack/daily-backup-status/action-buttons.jsx) component. We should make it optional, preferable in another PR.

Reply:
This has been addressed in a new PR: #77775